shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -136,24 +136,25 @@ private void doFallibleBridgeOperation(Runnable runnable,
         try {
             runnable.run();
         } catch (Exception e) {
+            LOGGER.warn("{} of Bridge '{}' failed", operation.getPrettyName(), bridge.getId(), e);
             bridge.setStatus(failedStatus);
             managerClient.notifyBridgeStatusChange(bridge)
                     .subscribe()
-                    .with(success -> LOGGER.debug(String.format("%s of Bridge '%s' failed", operation.getPrettyName(), bridge.getId()), e),
-                            failure -> failedToSendUpdateToManager(bridge, e));
+                    .with(success -> LOGGER.info("Failure notification for Bridge '{}' has been sent to the manager successfully", bridge.getId()),
```

Comment:
@manstis Actually the stacktrace is kept:
```
2022-07-18 11:21:52,047 INFO  [com.red.ser.sma.sha.ope.ManagerSyncServiceImpl] (vert.x-eventloop-thread-3) Provisioning notification for Bridge '932fa516-7049-481c-be51-cf221d4e607f' has been sent to the manager successfully
2022-07-18 11:21:52,047 INFO  [com.red.ser.sma.sha.ope.ManagerSyncServiceImpl] (vert.x-eventloop-thread-3) Provisioning Bridge '932fa516-7049-481c-be51-cf221d4e607f'
2022-07-18 11:21:52,106 WARN  [com.red.ser.sma.sha.ope.ManagerSyncServiceImpl] (vert.x-eventloop-thread-3) Provisioning of Bridge '932fa516-7049-481c-be51-cf221d4e607f' failed: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://192.168.39.53:8443/apis/com.redhat.service.bridge/v1alpha1/namespaces/ob-1608dc1f-4139-49c5-ae22-afe73ed807d7/bridgeingresses. Message: Forbidden! User minikube doesn't have permission. bridgeingresses.com.redhat.service.bridge "ob-932fa516-7049-481c-be51-cf221d4e607f" is forbidden: unable to create new content in namespace ob-1608dc1f-4139-49c5-ae22-afe73ed807d7 because it is being terminated.
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.requestFailure(OperationSupport.java:682)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.requestFailure(OperationSupport.java:661)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.assertResponseCode(OperationSupport.java:610)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:555)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:518)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:305)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:644)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:83)
	at io.fabric8.kubernetes.client.dsl.base.CreateOnlyResourceOperation.create(CreateOnlyResourceOperation.java:61)
	at io.fabric8.kubernetes.client.utils.CreateOrReplaceHelper.createOrReplace(CreateOrReplaceHelper.java:48)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.createOrReplace(BaseOperation.java:318)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.createOrReplace(BaseOperation.java:83)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.createOrReplace(BaseOperation.java:308)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.createOrReplace(BaseOperation.java:83)
	at com.redhat.service.smartevents.shard.operator.BridgeIngressServiceImpl.createBridgeIngress(BridgeIngressServiceImpl.java:70)
	at com.redhat.service.smartevents.shard.operator.BridgeIngressServiceImpl_Subclass.createBridgeIngress$$superforward1(Unknown Source)
	at com.redhat.service.smartevents.shard.operator.BridgeIngressServiceImpl_Subclass$$function$$5.apply(Unknown Source)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:54)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.proceed(InvocationInterceptor.java:62)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor.monitor(InvocationInterceptor.java:49)
	at io.quarkus.arc.runtime.devconsole.InvocationInterceptor_Bean.intercept(Unknown Source)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:41)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:41)
	at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:32)
	at com.redhat.service.smartevents.shard.operator.BridgeIngressServiceImpl_Subclass.createBridgeIngress(Unknown Source)
	at com.redhat.service.smartevents.shard.operator.BridgeIngressServiceImpl_ClientProxy.createBridgeIngress(Unknown Source)
	at com.redhat.service.smartevents.shard.operator.ManagerSyncServiceImpl.lambda$createBridgeIngress$16(ManagerSyncServiceImpl.java:118)
	at com.redhat.service.smartevents.shard.operator.ManagerSyncServiceImpl.doFallibleBridgeOperation(ManagerSyncServiceImpl.java:137)
	at com.redhat.service.smartevents.shard.operator.ManagerSyncServiceImpl.createBridgeIngress(ManagerSyncServiceImpl.java:118)
	at com.redhat.service.smartevents.shard.operator.ManagerSyncServiceImpl.lambda$doBridges$4(ManagerSyncServiceImpl.java:70)
......
2022-07-18 11:21:52,117 INFO  [com.red.ser.sma.sha.ope.ManagerSyncServiceImpl] (vert.x-eventloop-thread-3) Failure notification for Bridge '932fa516-7049-481c-be51-cf221d4e607f' has been sent to the manager successfully
```

Reply:
Logger process the vararg and if exception is detected then it is used to produce error message and stacktrace.

Reply:
+1 to this